docs/customizations.md

```diff
@@ -25,6 +25,21 @@ For instance the variable `proftpd_nat_masquerade` is set to `false` in `group_v
 apply the `[aws]` group variables which set `proftpd_nat_masquerade` to true, so that hosts in the aws group will have
 this aws-specific setting applied. Any combination of groups may be used.
 
+If you want to apply any of the changes you made to the variables you need to run the playbook again, making sure that
+the host you are targeting is in the right group. The simplest way to do so is to use an inventory file that only contains
+the group and the host you wish to target. If this is for example the group metavisitor, and you target the host localhost,
+your inventory file should look like this:
+
+```
+[metavisitor]
```

Comment:
Or we put it in examples or advanced examples. I mean vagrant is nice since you are not modifying the local environment, but there are some additional things you need to know, on top of all the other ansible-specific things :/